app/page.tsx

```diff
@@ -96,6 +96,7 @@ export default async function Page(props) {
         <ListItem href="/examples/post">[navigation] [blog post] [footer]</ListItem>
         <ListItem href="/examples/empty-application-template-page">Empty application template page</ListItem>
         <ListItem href="/examples/authentication">Authentication to API key</ListItem>
+        <ListItem href="/examples/web3-authentication">Authentication using Wallet Sign-in</ListItem>
```

Comment:
Add the vendor (we might have multiple, easier to search) I think its Third Web right?

Reply:
Added, and yes. Thirdweb